### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tadash10**
## Proposed changes

Hi there, I complied with the request they made to update the documentation #650 

## Types of changes

What types of changes does your code introduce to FESTIM?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [x] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [ ] Black formatted
- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by RemDelaporteMathurin — CHANGES_REQUESTED** [review]
Hi @tadash10 and welcome 🎉 😄 

Thanks for your PR! This will be needed for newcomers!

Here are a few suggestions:
- instead of describing the purpose of the PR in a comment, you can do it in the body of the PR under "Proposed changes".
- instead of copy pasting the issue, you can refer to it using its id like this #650 
- we should also add the discourse forum to the documentation page, in the [Where can I find help](https://festim.readthedocs.io/en/latest/userguide/beginners_guide.html) section. To do so, you can modify [this file](https://github.com/festim-dev/FESTIM/blob/main/docs/source/userguide/beginners_guide.rst)

**Comment by tadash10** [comment]
https://github.com/festim-dev/FESTIM/issues/650 update .Thanks a lot for the quick review and positive feedback!

File: docs/source/userguide/beginners_guide.rst [hunk 1]
```diff
@@ -12,10 +12,17 @@ Depending on how you install FESTIM, new users should familiarise with either `D
...
 ----------------------
 
 If you're having an issue, the best way to find help is to `raise an issue <https://github.com/RemDelaporteMathurin/FESTIM/issues/new/choose>`_ on the FESTIM GitHub repository.
 Before raising an issue, please check that someone didn't have the same problem to avoid duplicates.
 
+-----------------
+Discourse Forum
+-----------------
+
+Join the FESTIM community on our https://festim.discourse.group. Engage in discussions, ask questions, and connect with other users. This is a collaborative space where you can share your experiences and seek assistance.
```
Comment (RemDelaporteMathurin) [inline docs/source/userguide/beginners_guide.rst:28]:
```suggestion
----------------------
Where can I find help?
----------------------

If you're having an issue, the best way to find help is to join the `FESTIM discourse <https://festim.discourse.group>`. Engage in discussions, ask questions, and connect with other users. This is a collaborative space where you can share your experiences and seek assistance.
```

File: docs/source/userguide/beginners_guide.rst [hunk 2]
```diff
@@ -12,10 +12,10 @@ Depending on how you install FESTIM, new users should familiarise with either `D
...
+
+
 ----------------------
 Where can I find help?
 ----------------------
 
-If you're having an issue, the best way to find help is to `raise an issue <https://github.com/RemDelaporteMathurin/FESTIM/issues/new/choose>`_ on the FESTIM GitHub repository.
-Before raising an issue, please check that someone didn't have the same problem to avoid duplicates.
-
+If you're having an issue, the best way to find help is to join the `FESTIM discourse <https://festim.discourse.group>`. Engage in discussions, ask questions, and connect with other users. This is a collaborative space where you can share your experiences and seek assistance.
```
Comment (RemDelaporteMathurin) [inline docs/source/userguide/beginners_guide.rst:21]:
```suggestion
If you're having an issue, the best way to find help is to join the `FESTIM discourse <https://festim.discourse.group>`_. Engage in discussions, ask questions, and connect with other users. This is a collaborative space where you can share your experiences and seek assistance.
```

**Review by RemDelaporteMathurin — APPROVED** [review]

**Pull request merged**